explorer/pages/explore/[slug]/[...param].tsx

```diff
@@ -10,15 +10,24 @@ import { detail, listAll } from "../../../app/entity/api/service";
 import { getCurrentEntity } from "app/hooks/useCurrentEntity";
 import { DetailModel } from "../../../app/models/viewModels";
 import { Project } from "../../../app/views/Project/project";
+import { UUID_PARAM_INDEX } from "app/shared/constants";
 
 interface PageUrl extends ParsedUrlQuery {
   slug: string;
-  uuid: string;
+  param: string[];
 }
 
-const ProjectPage = (props: DetailModel): JSX.Element => {
+interface ProjectPageProps extends DetailModel {
+  slug: string;
+}
+
+const ProjectPage = ({ slug, ...props }: ProjectPageProps): JSX.Element => {
+  if (!slug) return <></>;
```

Comment:
This is a workaround that I couldn't find a better way to solve it (at least not yet). When building the project one of the pages being created doesn't have a slug, which will result in an error if I don't make this early return